rate-providers/InceptionLRTRateProvider_2.md

```diff
@@ -0,0 +1,79 @@
+# Rate Provider: `InwstETHsRateProvider`
+
+## Details
+- Reviewed by: @mkflow27
+- Checked by: @\<GitHub handle of secondary reviewer\>
+- Deployed at:
+    - [ethereum:0x3ba1a97D96F53611C4b2A788A5aa65c840d94c54](https://etherscan.io/address/0x3ba1a97D96F53611C4b2A788A5aa65c840d94c54)
+- Audit report(s):
+    - [InceptionLRT audits](https://docs.inceptionlrt.com/security/audit-reports)
+
+## Context
+The InceptionRateProvider is used to build a rate provider for inwstETHs LRT. It is a combination of two rate providers with an already reviewed one [Wsteth](https://etherscan.io/address/0x72D07D7DcA67b8A406aD1Ec34ce969c90bFEE768) and the [new one](https://etherscan.io/address/0xae48B92CBc0882a7D70D878e42cc121a62ceB632).  
+
+## Review Checklist: Bare Minimum Compatibility
+Each of the items below represents an absolute requirement for the Rate Provider. If any of these is unchecked, the Rate Provider is unfit to use.
+
+- [x] Implements the [`IRateProvider`](https://github.com/balancer/balancer-v2-monorepo/blob/bc3b3fee6e13e01d2efe610ed8118fdb74dfc1f2/pkg/interfaces/contracts/pool-utils/IRateProvider.sol) interface.
+- [x] `getRate` returns an 18-decimal fixed point number (i.e., 1 == 1e18) regardless of underlying token decimals.
+
+## Review Checklist: Common Findings
+Each of the items below represents a common red flag found in Rate Provider contracts.
+
+If none of these is checked, then this might be a pretty great Rate Provider! If any of these is checked, we must thoroughly elaborate on the conditions that lead to the potential issue. Decision points are not binary; a Rate Provider can be safe despite these boxes being checked. A check simply indicates that thorough vetting is required in a specific area, and this vetting should be used to inform a holistic analysis of the Rate Provider.
+
+### Administrative Privileges
+- [ ] The Rate Provider is upgradeable (e.g., via a proxy architecture or an `onlyOwner` function that updates the price source address).
+
+- [x] Some other portion of the price pipeline is upgradeable (e.g., the token itself, an oracle, or some piece of a larger system that tracks the price).
+    - upgradeable component: `RatioFeed` ([ethereum:0xFd73Be536503B5Aa80Bf99D1Fd65b1306c69B191](https://etherscan.io/address/0xFd73Be536503B5Aa80Bf99D1Fd65b1306c69B191#code))
+    - admin address: [ethereum:0x8e6C8799B542E507bfDDCA1a424867e885D96e79](https://etherscan.io/address/0x8e6C8799B542E507bfDDCA1a424867e885D96e79#readProxyContract3)
```

Comment:
Isn't this the address of the implementation contract owner? Looking into the ProxyAdmin owner, I found this address here: https://etherscan.io/address/0x650bD9Dee50E3eE15cbb49749ff6ABcf55A8FB1e#code
Which also seems to be multisig, but I didn't dig too deep there.

Reply:
You are correct that the `owner` of the ProxyAdmin is the `InceptionTimeLock`. The Timelock just acts as a passthrough with a delay. There are people allowed to schedule and execute upgrade proposals. The entity that is allowed to "go" or "no-go" the upgrade would be the `EXECUTOR` role. From that perspective this entity (The `EXECUTOR` role) is the "admin" in my opinion.